functionary/ui/templates/partials/modal.html

```diff
@@ -0,0 +1,17 @@
+<div class="modal is-active" _="on closeModal remove me">
+    <div class="modal-background" _="on click trigger closeModal"></div>
+    <div class="modal-card">
+        <header class="modal-card-head">
+            <h1 class="modal-card-title is-2">{{ title }}</h1>
```

Comment:
This would be more templatey as {% block motal_title %}{% endblock modal_title %}

Reply:
I think the goal I was going for with the partial that you just fill out parameters for was to enforce some sort of standard for how the forms get displayed. If I add blocks to the modals, that consistency could fade. In my mind, we should only be passing in a title, description, and the actual name of the partial to be rendered inside the modal body

Reply:
When I see blocks, I assume that the user is going to add additional styling or html tags within that block, which is what I want to prevent

Reply:
By just having the user fill out variables, my goal was to enforce the modal design at the implementation level, since the developer would not be inclined to add any formatting or extra html elements within the blocks. The developer would only need to worry about formatting their form in their own partial, and can just let the modal handle the rendering. There shouldn't be anything the developer needs to tweak within the modal. Nothing restricts the developer from putting the tags in the variable string, but they would have to actively go out of their way to do so.

I'm not particularly passionate about my implementation, I just believed mine would discourage any sort of modifications to the modal to keep everything consistent between the rendering of forms.

Reply:
I believe @kvig is correct that using blocks is more in line with the establish patterns of how to achieve what we're after here.  I don't hate the include implementation, but given that it's more consistent with django conventions and I think it actually solves some of the wonkiness with no longer being able to properly leverage the footer in the modals, I'm going to say let's switch to using blocks rather than the include style.

Reply:
Fixed. Not too sure I'm satisfied with the way I implemented it, but let me know your thoughts on it